src/main/java/org/asamk/signal/json/JsonAttachmentData.java

```diff
@@ -0,0 +1,3 @@
+package org.asamk.signal.json;
+
+public record JsonAttachmentData(String dataBase64) {}
```

Comment:
A future performance improvement for larger attachments might be to use an InputStream and a custom jackson serializer here to not load the complete file in memory.

Reply:
That was actually pretty straight forward to implement so I did that.